### ISSUE TITLE
APT upgrades to wrong versions of mesa libraries, rendering Steam, Lutris and potentially others unusable

### ISSUE THREAD

**Issue by Spicysauced** [pop-os/pop#1607]
**Distribution (run `cat /etc/os-release`):** 20.10, Kernel 5.11
 
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):** Dont know, updater? Apt? Server? Repositories?

**Issue/Bug Description:** 
After the recent Mesa and Kernel update, package management broke. 
After a restart after installing the new 5.11 Kernel, more updates were displayed, see below.

apt list --upgradable says:

```
libegl-mesa0/groovy 21.0.0-1pop0~1616205663~20.10~73b50a1 amd64 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
libgbm1/groovy 21.0.0-1pop0~1616205663~20.10~73b50a1 amd64 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
libgl1-mesa-dri/groovy 21.0.0-1pop0~1616205663~20.10~73b50a1 amd64 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
libgl1-mesa-dri/groovy 20.3.2-1pop1~1612283689~20.10~c8aa2e6 i386 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
libglapi-mesa/groovy 21.0.0-1pop0~1616205663~20.10~73b50a1 amd64 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
libglapi-mesa/groovy 20.3.2-1pop1~1612283689~20.10~c8aa2e6 i386 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
libglx-mesa0/groovy 21.0.0-1pop0~1616205663~20.10~73b50a1 amd64 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
libglx-mesa0/groovy 20.3.2-1pop1~1612283689~20.10~c8aa2e6 i386 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
libosmesa6/groovy 21.0.0-1pop0~1616205663~20.10~73b50a1 amd64 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
mesa-vulkan-drivers/groovy 21.0.0-1pop0~1616205663~20.10~73b50a1 amd64 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
mesa-vulkan-drivers/groovy 20.3.2-1pop1~1612283689~20.10~c8aa2e6 i386 [aktualisierbar von: 20.2.6-0ubuntu0.20.10.1]
```

Maybe you can spot the issue already. I couldnt at first, as I am not that experienced.
After doing these upgrades, Steam started complaining, and Lutris didnt find any 32 bit Vulkan drivers anymore, leading to error messages. 

Through googling I found out, that if ONE version of a 32 and 64 bit library needs to be installed at the same time, they also need to be "in sync", aka on the **same version** (referenced here: https://github.com/ValveSoftware/steam-for-linux/issues/7282#issuecomment-661815885), which wouldnt be the case anymore if I updated. 

Example: libgl1-mesa-dri 64 bit updates correctly to 21.0.0-1, whereas 32 bit updates to 20.3.2-1. 
That is the culprit and breaks Mesa. 
I definitely either need a bugfix and/or assistance with that. 
Why is this happening?! I dont have any weird PPAs enabled, just Steam itself and Chrome. 

**Steps to reproduce (if you know):**
Actually no idea. The same updates happened on my wife's PC with no issues, also on the same Pop version.

**Expected behavior:**
All 32/64 bit packages update to the same version.

**Other Notes:**

**Comment by Spicysauced** [comment]
Solved!

The apt cache got corrupted somehow. This is the last idea I had at 5am...
I figured that the servers cant be the issue since as mentioned my wife's PC upgraded properly. So the only things that could have happened were either a program preventing the upgrade of said packages (unlikely, since I dont have a lot of apps), or that apt itself got messed up.

The solution:
`sudo rm -r /var/cache/apt /var/lib/apt/lists` and
`sudo apt update`

Then, to be sure about whats going to happen another
`apt list --upgradable`, which showed all the correct versions for all packages (21.0.0-1)

At last: `sudo apt upgrade`

.. just took me 12 hours as a noob to figure this out. No idea how that could happen really.
